spec/features/reservations_spec.rb

```diff
@@ -545,4 +545,24 @@
       end
     end
   end
+  context 'redirects properly' do
```

Comment:
This message should be a bit more specific, maybe 'accessing new_reservations_path with empty cart', and then each test would describe what they're testing (e.g. 'handles direct url visit' and 'handles reserve button click' or something like that).